### DISCUSSION TITLE
Gyro only works in Cemu when the Quick Access menu is displaying?

### DISCUSSION THREAD

**Discussion by Ruharc** [kmicki/SteamDeckGyroDSU#48] [Q&A]
This is a wierd problem I am having.  I installed using EmuDeck, and am using Cemu 2.x.  I have installed SteamDeckGyroDSU using both EMUdeck and the Github install instructions.  When I load up Cemu and BotW the gyro only works when aiming if I click the Quick Access menu button then the gyro works in game correctly with the menu displaying on the side.

I have verified all of the settings in CEMU for input and it is all set up correctly as directed here.  I have ran the commands to verify the server is running and it is running correctly.

Any ideas?

Thanks everyone!

**Comment by kmicki** [comment]
It's a problem with a new Steam Beta (maybe stable too now, I don't know). See issue #45.
